docs/phdi-fhir-api-template.postman_collection.json

```diff
@@ -0,0 +1,473 @@
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{FHIR-server-url}}/Patient",
+					"host": [
+						"{{FHIR-server-url}}"
+					],
+					"path": [
+						"Patient"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Patient",
+			"request": {
+				"method": "DELETE",
+				"header": [],
+				"url": {
+					"raw": "{{FHIR-server-url}}/Patient/{{deleted-patient-id}}",
+					"host": [
+						"{{FHIR-server-url}}"
+					],
+					"path": [
+						"Patient",
+						"{{deleted-patient-id}}"
+					]
+				}
+			},
+			"response": []
+		}
+	],
+	"auth": {
+		"type": "bearer",
+		"bearer": [
+			{
+				"key": "token",
+				"value": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ii1LSTNROW5OUjdiUm9meG1lWm9YcWJIWkdldyIsImtpZCI6Ii1LSTNROW5OUjdiUm9meG1lWm9YcWJIWkdldyJ9.eyJhdWQiOiJodHRwczovL3RhYnVsYXRpb25kZWJ1Zy1zZXJ2ZXIxLmZoaXIuYXp1cmVoZWFsdGhjYXJlYXBpcy5jb20iLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC8yOGNmNThkZi1lZmU4LTQxMzUtYjJkMS1mNjk3ZWU3NGMwMGMvIiwiaWF0IjoxNjc1MTkwOTc1LCJuYmYiOjE2NzUxOTA5NzUsImV4cCI6MTY3NTE5NDkzMSwiYWNyIjoiMSIsImFpbyI6IkFWUUFxLzhUQUFBQVNxcFpJY3ByZnlGelV4a3k5dXJWeEIzeFRuakJPZWY1VzJHb2I4MjY1TEt3R25Uc3h4RkhQL1VGL0VwcEMrVVFqUVZJNkFJaWl4RW5pNzNpREowQmdvaUt5UHUyK3lVajZmcTZybUdPWGY4PSIsImFtciI6WyJwd2QiLCJtZmEiXSwiYXBwaWQiOiIwNGIwNzc5NS04ZGRiLTQ2MWEtYmJlZS0wMmY5ZTFiZjdiNDYiLCJhcHBpZGFjciI6IjAiLCJmYW1pbHlfbmFtZSI6IkZhcnJlbGwiLCJnaXZlbl9uYW1lIjoiR29yZG9uIiwiaXBhZGRyIjoiNjcuMTY2LjI2LjgwIiwibmFtZSI6IkdvcmRvbiBGYXJyZWxsIiwib2lkIjoiODViMmVlYTAtODJjMC00NThlLWFlMzgtYzZmMGY4NGZjYjEwIiwicHVpZCI6IjEwMDMyMDAyNTJFNjJENDQiLCJyaCI6IjAuQVVVQTMxalBLT2p2TlVHeTBmYVg3blRBRE5oNFowX3ZXdHhEb2Ytd2MzSkxsSlZGQUIwLiIsInNjcCI6InVzZXJfaW1wZXJzb25hdGlvbiIsInN1YiI6IkpReGw0a1Vqa09sZVBVc3RwaXV3Rm5qbFFybWtYS3RrWEV4X2FZN1FFaWsiLCJ0aWQiOiIyOGNmNThkZi1lZmU4LTQxMzUtYjJkMS1mNjk3ZWU3NGMwMGMiLCJ1bmlxdWVfbmFtZSI6ImdvcmRvbkBza3lsaWdodGhxLm9ubWljcm9zb2Z0LmNvbSIsInVwbiI6ImdvcmRvbkBza3lsaWdodGhxLm9ubWljcm9zb2Z0LmNvbSIsInV0aSI6IjFpRWp3c1JZOTBlbVlIZFgxd2NwQUEiLCJ2ZXIiOiIxLjAifQ.BnG5uGD59X7QjIZzeFoNR9KeMLoW3nVpJULBmXxNkBX6od7NlQhd7v9GB4n4C0d7_LxBD_oi8I2u6Q5sgUNXha6OrqkGOu1ZeIWXxo7WePkXCmxi13FdsTG3KPPxafq2DeZVjLDIYNft8Je1NWhNLkYLXExOF-WUKQxB0x4j55YI8lr4JBjbA4OR62qWkC2o5HO3jJ8ThBFI-pAKmFm1TebKx_v94XsIACTll_Q2B3WnuSqxFlC3oFLuklInV3UkS14Cbc-gJRip26r7Z6bzFur1KNMx774YaYyYn2MC9yt6d01OyderhqX8Xa9RYH3LNCKeml6XvO1aU9QQyZnsVQ",
+				"type": "string"
+			}
+		]
+	},
+	"event": [
+		{
+			"listen": "prerequest",
+			"script": {
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		},
+		{
+			"listen": "test",
+			"script": {
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		}
+	],
+	"variable": [
+		{
+			"key": "FHIR-server-url",
+			"value": "https://tabulationdebug-server1.fhir.azurehealthcareapis.com",
```

Comment:
should probably also clear the values for all these variables too before merging, unless we're gonna have folks test with this fhir server specifically.